src/base/bittorrent/session.cpp

```diff
@@ -537,7 +532,15 @@ Session::Session(QObject *parent)
     m_resumeDataSavingManager->moveToThread(m_ioThread);
     connect(m_ioThread, &QThread::finished, m_resumeDataSavingManager, &QObject::deleteLater);
     m_ioThread->start();
-    m_resumeDataTimer->start();
+
+    // Regular saving of fastresume data
+    m_resumeDataTimer = new QTimer(this);
+    connect(m_resumeDataTimer, &QTimer::timeout, this, [this]() { generateResumeData(); });
```

Comment:
Can't remember, why it isn't `connect(m_resumeDataTimer, &QTimer::timeout, this, &Session::generateResumeData);`...

Reply:
compiler doesn't accept it because of signature mismatch.